kani-driver/src/args/mod.rs

```diff
@@ -177,6 +177,10 @@ pub struct VerificationArgs {
     )]
     pub harnesses: Vec<String>,
 
+    /// If specified, with the fully qualified name of the harness provided, runs that specific harness and nothing more
```

Comment:
What about: "When specified, the harness filter will only match the exact fully qualified name of a harness"

Reply:
Can you also please add a note to the `harness` documentation saying that `--exact` will change its behavior?

Reply:
By documentation, I'm assuming you mean the doc comment written in the same struct above? Can do